tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -334,14 +334,22 @@ public function test_sync_site_fields() {
 		$this->assertEquals( 'POST', $this->request_args['method'] );
 		$this->assertEqualSets(
 			array(
-				'site_id',
-				'site_secret',
-				'url',
+				'action_uri',
+				'analytics_redirect_uri',
+				'application_name',
+				'hl',
+				'mode',
 				'name',
+				'nonce',
 				'redirect_uri',
 				'return_uri',
-				'action_uri',
-				'analytics_redirect_uri',
+				'scopes',
+				'service_version',
+				'site_id',
+				'site_secret',
+				'supports',
+				'url',
+				'user_roles',
 			),
 			array_keys( $this->request_args['body'] )
 		);
```

Comment:
While this would be somewhat similar, could you also add one simple test for the `register_site` method?

Reply:
Added.